cargo-dist-schema/src/lib.rs

```diff
@@ -6,7 +6,13 @@ use std::path::PathBuf;
 /// The final report of cargo-dist
 #[derive(Debug, Clone, Serialize, Deserialize, JsonSchema)]
 pub struct DistManifest {
+    /// The version of cargo-dist that generated this
```

Comment:
It Shouldn't Be Optional but it just got added so everything will explode if it's not (e.g. if Oranda consumes the next release's dist-manifest.json which still won't have this field because Bootstrapping Is Real And Your Enemy).

In later releases we can make it non-optional (effectively deprecating the old prereleases).